BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -222,8 +212,14 @@ fileprivate final class ScrollerWrapperView: UIView {
             self.scrollView.alwaysBounceHorizontal = scrollView.alwaysBounceHorizontal
         }
 
-        if self.scrollView.alwaysBounceVertical != scrollView.alwaysBounceVertical {
```

Comment:
May want to preserve the logic that prevents setting this property to the same value.